### PR TITLE
treewide: replace AUTORELEASE/COMMITCOUNT with real PKG_RELEASE

### PR DESCRIPTION
**Pull request by 1715173329**
Based on Paul Fertser \<fercerpav@gmail.com\>'s guidance:
Change AUTORELEASE in `rules.mk` to:
```
AUTORELEASE = $(if $(DUMP),0,$(shell sed -i "s/\$$(AUTORELEASE)/$(call commitcount,1)/" $(CURDIR)/Makefile))
```

then update all affected packages by:
```
for i in $(git grep -l PKG_RELEASE:=.*AUTORELEASE | sed 's^.*/\([^/]*\)/Makefile^\1^';);
do
	make package/$i/clean
done
```

### CODE REVIEW

**Comment by 1715173329** [comment]
cc @aparcar

File: package/base-files/Makefile [hunk 1]
```diff
@@ -13,7 +13,7 @@ include $(INCLUDE_DIR)/feeds.mk
 
 PKG_NAME:=base-files
 PKG_FLAGS:=nonshared
-PKG_RELEASE:=$(COMMITCOUNT)
+PKG_RELEASE:=1530
```
[the remark this answers is not in the document]
Comment (1715173329) [reply package/base-files/Makefile]: done ;)

**Comment by hnyman** [comment]
The commit title and the commit message talk about PKG_VERSION, while you actually touch PKG_RELEASE...

**Comment by 1715173329** [comment]
> The commit title and the commit message talk about PKG_VERSION, while you actually touch PKG_RELEASE...

Oh yes, will correct soon, thank you!

**Comment by aparcar** [comment]
Thanks!

**Pull request closed without merging**